validators/contactsValidator.js

```diff
@@ -0,0 +1,46 @@
+const joi = require("joi");
+
+const newContactValidator = joi.object({
+	name: joi
+		.string()
+		.regex(/^[a-zA-Z\s]+$/)
+		.required(),
+
+	email: joi
+		.string()
+		.email({
+			minDomainSegments: 2,
+		})
+		.required(),
+
+	phone: joi
+		.string()
+		.regex(/^[\d+\-()\s]{9,15}$/)
+		.required(),
```

Comment:
To sprawia że pole nie może być undefined :)

Reply:
A może być?